src/utilities/ScreenProvider/index.jsx

```diff
@@ -0,0 +1,66 @@
+/* global window */
+/* eslint no-console: off */
```

Comment:
Why do you need it here?

Reply:
I never used eslint and don't intend to in the future. This file was based off of another local source file in the repo.